docs/API.md

```diff
@@ -64,3 +64,29 @@ Acceptable formatting:
 
 ### The body of your document
 (required) A longer description of the topic, which will appear on its topic page. Must be 1,000 characters or less. Should not be the same as `short_description`. Some Markdown is allowed, such as links. Emoji are allowed.
+
+##  Making edits to a collection
+
+If you're making edits to a collection, the following fields are available for use. Not all fields are required.
+
+### items
+A comma-separated string containing any of the following values:
+
+- GitHub repository path (e.g. `defunkt/dotjs`)
+- GitHub username (e.g. `defunkt`)
+- GitHub organization (e.g. `github`)
+- Any web URL (e.g. `https://product.hubspot.com/blog/git-and-github-tutorial-for-beginners`)
+- Any YouTube video URL (e.g. `https://www.youtube.com/watch?v=0fKg7e37bQE`)
+
+A complete `items` key-value pair might look like:
+
+`items: pybee/batavia, Homebrew/brew, https://www.youtube.com/watch?v=dSl_qnWO104`
+
+### created_by
+(if applicable) GitHub username of the person and/or organization that authored the collection.
+
+### display_name
+(required) The collection name that will be displayed on its page (ex. `How to choose (and contribute to) your first open source project`). Should use proper noun capitalization. Emoji are not allowed.
```

Comment:
> Emoji are not allowed.

😭

Reply:
@talsafran , @damaneice: Should I create an issue to build emoji support in display names in the future?

Reply:
hm.. I would be cool with creating an issue for it.

Reply:
Yah same. Or punt 👢?

Not sure how critical a requirement it is. Fwiw, topics don't currently have emoji support.